doc/nrf/protocols/wifi/advanced_modes/mem_requirements_raw.rst

```diff
@@ -0,0 +1,26 @@
+.. _ug_wifi_mem_req_raw_mode:
+
+Memory requirements for Wi-Fi applications in Raw mode
+######################################################
+
+Code and RAM memory footprint requirements differ depending on the selected platform and the application example.
+
+The following table list memory requirement values for selected Wi-Fi® samples supporting Raw mode.
+
+Footprint values are provided in kilobytes (KB).
+
+.. tabs::
+
+   .. tab:: nRF7002 DK
+
+      The following table lists memory requirements for samples running on the :ref:`nRF7002 DK <programming_board_names>` (:ref:`nrf7002dk/nrf5340/cpuapp <nrf7002dk_nrf5340>`).
+
+      +----------------------------------------------------+-------------+-------------------------------------------+----------------------------+----------------------+---------------------------------+--------------------+----------------------+
+      | Sample                                             |   Total ROM |   Wi-Fi driver ROM                        |         nRF70 FW patch ROM |   WPA supplicant ROM |   Total RAM (incl. static heap) |   Wi-Fi driver RAM |   WPA supplicant RAM |
+      +====================================================+=============+===========================================+============================+======================+=================================+====================+======================+
+      | :ref:`Raw TX packet <wifi_raw_tx_packet_sample>`   |         507 |                                        54 |                         69 |                  180 |                             298 |                207 |                   13 |
+      +----------------------------------------------------+-------------+-------------------------------------------+----------------------------+----------------------+---------------------------------+--------------------+----------------------+
+      | :ref:`Monitor <wifi_monitor_sample>`               |         495 |                                        51 |                         69 |                  180 |                             398 |                306 |                   13 |
```

Comment:
WPA supplicant isn't involved in monitor mode, I have raised it with the team, a PR will be raised to fix this and then we can update this row.

Reply:
I think we can update the values in the doc improvement phase after this PR is merged.